Documentation/operations/upgrade.rst

```diff
@@ -188,19 +195,18 @@ command line or by committing the values to a YAML file:
    options, either by setting them at the command line or storing them in a
    YAML file, similar to:
 
-   .. .. parsed-literal::
-
-    agent: true
-    keepDepreatedProbes: true
+   .. code-block:: yaml
 
+      agent: true
+      upgradeCompatibility: 1.7
 
-   You can then pass the values file to Helm by running:
+   You can then upgrade using this values file Helm by running:
```

Comment:
"values file Helm" reads strange to me.
Is any of "values Helm file" or "Helm-values file" better?

Reply:
Actually, I think it should be `Helm values file` or `Helm values-file`. Or `Helm-values file` as you said if we think this is a file of Helm values, but that didn't seem to be the original meaning.

Maybe just better to drop `Helm` from the sentence :smile: